scanner.py

```diff
@@ -0,0 +1,45 @@
+# References: 
```

Comment:
You may also place the references in your README so it won't take up too much lines of code in your script.